hieradata_aws/class/draft_frontend.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 
+govuk::apps::collections::memcache_servers: '127.0.0.1:11211'
```

Comment:
What does it mean for this to be set to localhost on the draft app? Is it really just a dummy value? Do you intend for memcache to be disabled for draft?

Similarly below with the default on the arg to the class - I'm not clear as to the intent and the meaning.

Reply:
Good point. It refers to the instance of memcache on the local machine (one is installed by puppet). I wonder if it'll be cleaner if it's just undefined though and the machine picks it up automatically (as it does currently)

Reply:
That seems much clearer to me, thanks!